ci-tests/jest-image-snapshot/tests/tests.spec.js

```diff
@@ -33,9 +33,16 @@ describe('dsop-kontroll', () => {
             await takeSnapshot(`saksoversikt-${name}`, page);
         });
 
-        test('opprett-behandling', async () => {
+        test('opprett-fagsak', async () => {
             await page.goto('http://ci-test-server:8000/fagsak/opprett');
-            await page.waitFor('.opprett');
+            await page.waitFor('.opprettfagsak');
+            await page.waitFor(1000);
+            await takeSnapshot(`opprett-fagsak-${name}`, page);
+        });
+
+        test('opprett-behandling', async () => {
+            await page.goto('http://ci-test-server:8000/fagsak/1/ny-behandling');
+            await page.waitFor('.opprettbehandling');
             await page.waitFor(1000);
             await takeSnapshot(`opprett-behandling-${name}`, page);
```

Comment:
Ikke så viktig altså, men er det en grunn til at vi skiller på opprett og ny? I api bruker vi jo ny-fagsak. Kunne man benytta kun den ene når det dreier seg om å opprette ny? :)

Reply:
Yes, kan endre 👍

Reply:
( jeg stemmer i så fall for å benytte "opprett" i api også, men har ikke sterke følelser rundt det )

Reply:
Enig! Endrer overalt 👍